src/components/link/link.jsx

```diff
@@ -1,21 +1,17 @@
 import React from 'react';
 import cx from 'classnames';
 import PropTypes from 'prop-types';
-import { BEMClass } from '../../helpers/utils/bem-helper';
 import CoreLink from '../../helpers/components/Link/Link';
-import './link.scss';
-
-const BASE_CLASS = 'vibe-sb-comps-link';
-const bemHelper = BEMClass(BASE_CLASS);
+import styles from './link.scss';
```

Comment:
Only forgot to rename imported file - should be `link.module.scss`

Reply:
done👍